src/pages/Delivery/Delivery.layout.jsx

```diff
@@ -1,23 +1,15 @@
 /* eslint-disable no-unused-vars */
 // @ts-nocheck
 import PropTypes from 'prop-types'
-import { Text } from '../../components'
-import {
-  MapLayoutContainer,
-  HeaderMap,
-  HeaderText,
-  ChatBtnContainer,
-  ChatBtn,
-  HeaderTitle,
-  HeaderSubTitle,
-  FooterMap,
-  FooterMapAddress,
-  MapContainer,
-} from './Delivery.styled'
+import { Fragment } from 'react'
 import assistant from '../../assets/assistant.png'
 import chat from '../../assets/chat.png'
 import gps from '../../assets/gps-icon.png'
-import { Fragment } from 'react'
+import { Text } from '../../components'
+import {
+  ChatBtn, ChatBtnContainer, FooterMap,
+  FooterMapAddress, HeaderMap, HeaderSubTitle, HeaderText, HeaderTitle, MapContainer, MapLayoutContainer
+} from './Delivery.styled'
```

Comment:
Please check if you can run Prettier or similar to sort correctly the imports, I think is more readable one on top of the other